testsuite/features/support/setup_browser.rb

```diff
@@ -20,8 +20,8 @@
 $stdout.sync = true
 STARTTIME = Time.new.to_i
 Capybara.default_max_wait_time = 10
-DEFAULT_TIMEOUT = 250
-CLICK_TIMEOUT = Capybara.default_max_wait_time * 2
+DOUBLE_TIMEOUT = Capybara.default_max_wait_time * 2
+BIG_TIMEOUT = 250
```

Comment:
The default timeout, the timeout got by all Capybara methods if you don't specify something different is `Capybara.default_max_wait_time`, so I would like to avoid having two variables defining "default" timeout, as it was confusing.

The current `DEFAULT_TIMEOUT`was, in fact, a huge value used in some case, to wait long enough, but is not the default value used in Capybara action, so my suggestion is to rename it with something different, if `BIG_TIMEOUT` doesn't look good, we can rename to something different.

The `CLICK_TIMEOUT` was a timeout a bit bigger than the default timeout in Capybara, so when we performed clicks that triggers some refresh in the page, we could wait enough time to have the page loaded, but recently we used that variable in places that are not specifically a click, that was the reason to rename it to a `DOUBLE_TIMEOUT`, we can rename it to something different.